src/pythie_serving/run.py

```diff
@@ -5,14 +5,29 @@
 from logging.config import dictConfig
 
 from google.protobuf import text_format
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.instrumentation.grpc import GrpcInstrumentorServer
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
 
 from pythie_serving import create_grpc_server
 from pythie_serving.tensorflow_proto.tensorflow_serving.config import (
     model_server_config_pb2,
 )
 
 
+def initialize_tracing():
+    tracing_collector_host = str(os.environ.get("TRACING_COLLECTOR_HOST"))
```

Comment:
* OPENTELEMETRY_COLLECTOR_HOST maybe?
* you'll need to setup the variable in the deployment chart, here: https://github.com/wiremind/wiremind-helm-charts/blob/main/charts/pythie/templates/pythie-serving-deployment.yaml#L95
* finally, update the readme please to explain the setup